gui/canvaspainter/v7/qt5/rootwebview.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RootWebView_h
+#define RootWebView_h
+
+#include <QWebEngineView>
+#include <QWebEnginePage>
+
+
+class RootWebPage : public QWebEnginePage {
```

Comment:
It is not possible - I need to generate Qt dictionary (MOC file) for qt classes to use signal-slot for them. Instead, I extract this class in separate header and source files